src/components/Publishing/PublishingProposalsList/PublishingProposalsListItem.vue

```diff
@@ -129,6 +137,14 @@ export default {
     owner: function() {
       return this.proposal.ownerName
     },
+    //proposer's email will render if the email is provided adn not an empty string
+    email() {
+      if (this.proposal.emailAddress && this.proposal.emailAddress.trim() !== ''){
```

Comment:
Heads up, in JS you have to write this like:

`this.proposal.emailAddress !== '' && this.proposal.emailAddress.trim() !== ''`

if you want to check that both sides of the condition `&& `are `!== ''` - the way it's written here, it only checks that the righthand side of the && is !== to '' 

I'd recommend writing it like this instead because it's easier to read: 

this.proposal.emailAddress.length && this.proposal.emailAddress.trim().trim.length

This will assess as true if both sides of the condition are true, and false if either side is an empty string.

Reply:
Thank you! Just changed